src/EnergyPlus/SetPointManager.cc

```diff
@@ -8321,7 +8319,7 @@ void DefineReturnWaterHWSetPointManager::calculate(EnergyPlusData &state, DataLo
     // calculate the supply setpoint to use, default to the design value if flow is zero
     Real64 T_supply_setpoint = this->maximumHotWaterSetpoint;
     if (mdot > DataConvergParams::PlantFlowRateToler) {
-        T_supply_setpoint = T_return_target + Qdemand / (mdot * cp);
+        T_supply_setpoint = T_return_target + deltaT;
```

Comment:
Using `Qdemand / (mdot * cp) `means just using `deltaT`

So we do not need the call to GetSpecificHeatGlycol nor the calculation of Qdemand